src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -97,6 +98,10 @@ export default {
 		tooltip: Tooltip,
 	},
 
+	mixins: [
+		participant,
+	],
+
 	components: {
```

Comment:
GitHub Actions / eslint node12.x

src/components/MessagesList/MessagesGroup/Message/Message.vue#L105

The "components" property should be above the "mixins" property on line 101

Reply:
adjusted